packages/playground/data-liberation/src/stream-api/WP_File_Byte_Stream.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+class WP_File_Byte_Stream extends WP_Byte_Stream {
+
+	protected $file_path;
+	protected $chunk_size;
+	protected $file_pointer;
+	protected $offset_in_file;
+
+	public function __construct( $file_path, $chunk_size = 8096 ) {
```

Comment:
I'm curious, where is the _8096_ chunk size coming from?

Reply:
It comes out of a hat :D good spot, we could use pretty much any number here and perhaps 32k, 64k, or even 512k would work better

Reply:
but! I'd add an issue to revisit this later on and stick to a tiny chunk size for now, perhaps even lower it to 100, just to be exposed to all the streaming-related problems very early

Reply:
documented here https://github.com/WordPress/wordpress-playground/issues/1957